src/OrleansServiceBus/Providers/Streams/EventHub/EventHubDataAdapter.cs

```diff
@@ -237,7 +235,7 @@ protected virtual IBatchContainer GetBatchContainer(EventHubMessage eventHubMess
         /// <returns></returns>
         public virtual StreamSequenceToken GetSequenceToken(ref CachedEventHubMessage cachedMessage)
         {
-            return new EventSequenceTokenV2(cachedMessage.SequenceNumber, 0);
+            return new EventHubSequenceTokenV2("", cachedMessage.SequenceNumber, 0);
```

Comment:
Some static analysis tools will complain if `""` is used instead of `string.Empty`.